docs/getting-started/live-networks.md

```diff
@@ -10,10 +10,10 @@ By default, the `regen` binary will connect to a local node. In order to connect
 
 `regen-1` is the chain ID for Regen Mainnet.
 
-<!-- TODO: add information about genesis binary and upgrade binaries -->
-
 Regen Mainnet launched with the `v1.0.0` release tag of `regen-ledger`.
 
+When starting a full node or a validator node from genesis, you will need to start the node with the `v1.0.0` version (the "genesis binary"). For more information about preparing your node to migrate to the `v2.0.0` version, see [Upgrade Guide v2.0](../migrations/v2.0-upgrade.md).
```

Comment:
We should at some point follow this up with instructions to use [state sync](https://blog.cosmos.network/cosmos-sdk-state-sync-guide-99e4cf43be2f) instead.

Reply:
Just wrote this up in #625 so we don't forget